app/app.js

```diff
@@ -18,15 +17,22 @@ function Chicken (chix)  {
   };
 }
 
+Chicken.all = [];
 //passes constructed objects to allChickens
-rawChickenData.forEach(function(ele){
-  allChickens.push(new Chicken(ele));
-});
+// rawChickenData.forEach(function(ele){
+//   allChickens.push(new Chicken(ele));
+// });
 
 //sorts allChickens title alphabeticaly
-allChickens.sort(function(a,b){
+Chicken.loadAll = function (rawChickenData){
+rawChickenData.sort(function(a,b){
   return (a.title > b.title);
 });
+rawChickenData.forEach(function(ele){
+  Chicken.all.push(new Chicken(ele));
+})
+}
+
```

Comment:
Pay attention to indenting: it makes your code easier to read, not just for others, but for you at a later time.